themes/custom/droopler_theme/js/global.js

```diff
@@ -62,4 +62,10 @@
     }
   };
 
+  Drupal.behaviors.droopler_unpublished = {
+    attach: function (context, settings) {
+      $("<div class=\"unpublished-message\">Unpublished</div>").insertBefore(".blog_post_header");
```

Comment:
not sure if this is right way
- code always insert this text before css class blog_post_header event its not unpublished?
- is text translatable?

cant we